spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -221,6 +255,69 @@ public WebServer getWebServer(HttpHandler httpHandler) {
         return armeriaWebServer;
     }
 
+    private static List<ServerPort> armeriaPorts(ServerBuilder sb) {
+        try {
+            final Field ports = ServerBuilder.class.getDeclaredField("ports");
+            ports.setAccessible(true);
+            @SuppressWarnings("unchecked")
+            final List<ServerPort> armeriaPorts = (List<ServerPort>) ports.get(sb);
+            return armeriaPorts;
+        } catch (NoSuchFieldException | IllegalAccessException ignored) {
+            throw new Error(); // Should never reach here.
```

Comment:
We may get `IllegalAccessException` in some cases. How about throwing an `IllegalStateException` with a proper message and a cause?

Reply:
Thanks fixed. 😄